cmd/run.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"github.com/astronomer/astro-cli/cmd/utils"
+	"github.com/astronomer/astro-cli/config"
+	"github.com/spf13/cobra"
+)
+
+var (
+	dagID     string
+	startDate string
+	taskLogs  bool
+)
+
+func newRunCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "run DAG-ID",
+		Short: "Run a local DAG withpython by running its tasks sequentially",
+		Long:  "Run a local DAG by running its tasks sequentially. This command will spin up a docker airflow environment and execute your DAG code",
+		Args:  cobra.MaximumNArgs(1),
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+		PreRunE: utils.EnsureProjectDir,
+		RunE:    run,
+	}
+	cmd.Flags().StringVarP(&envFile, "env", "e", ".env", "Location of file containing environment variables")
+	cmd.Flags().BoolVarP(&noCache, "no-cache", "", false, "Do not use cache when building container image")
+	cmd.Flags().BoolVarP(&taskLogs, "task-logs", "", false, "Show task logs while the DAG is running")
+	cmd.Flags().StringVarP(&settingsFile, "settings-file", "s", "airflow_settings.yaml", "Settings or env file export objects too")
+	cmd.Flags().StringVarP(&startDate, "start-date", "d", "", "The start date for the DAG this is the current date and time by default")
```

Comment:
is `startDate` being used anywhere? can't see any reference of it being used

Reply:
startDate feature doesn't exist yet

Reply:
I'll remove since it seems it will be awhile till we have it